tools/lib/getDirname.js

```diff
@@ -0,0 +1,9 @@
+import { fileURLToPath } from "node:url";
+import path from "path";
```

Comment:
Minor, but might as well make this "node:path"

Reply:
Done & done 😄